clients/lsp-sqls.el

```diff
@@ -52,23 +52,124 @@
     ;; we can add some options to command. (e.g. "-config")
     base))
 
+
+(defcustom lsp-sqls-timeout 0.5
+  "Timeout to use for `sqls' requests"
+  :type 'number
+  :package-version '(lsp-mode . "7.1.0"))
+
+(defcustom lsp-sqls-connections nil
+  "The connections to the SQL server(s)."
+  :type '(repeat (alist :key-type (choice
+                                   (const :tag "Driver" driver)
```

Comment:
Don't you think that this should be an alist `((DRIVER . CONNECTION-STRING)...)` instead? (i.e. `(("sqlite" . "$PATH_TO_DB"))`). This would have the benefit of reducing typing and eliminating the possibility of misspelling.

Disregard this advice if there are other valid keys.

Reply:
> Disregard this advice if there are other valid keys.

I don't know - but in the future, the author might add new properties, which will force us to make backward-incompatible changes.